openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +157,8 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        self.add_resource_urls(fragment)
+
         course = self.runtime.modulestore.get_course(self.course_key)
```

Comment:
This wasn't your change, but it seems expensive to have to get the whole course just to render this one XBlock. In looking at the template, it doesn't seem like the course context variable is used, but it is hard to tell due to template inheritance. Could you try removing it and see if it is necessary. Hopefully we don't force XBlocks to have to know how to do this.